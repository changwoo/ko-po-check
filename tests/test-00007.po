# -*-text-*-
# test 7 - GNOME
#
msgid ""
msgstr ""
"Project-Id-Version: test 7\n"
"POT-Creation-Date: 2002-08-01 01:30+0900\n"
"PO-Revision-Date: 2002-08-01 01:30+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=EUC-KR\n"
"Content-Transfer-Encoding: 8bit\n"

# E: 번역자 써야 함
#: test.c:7
msgid "translator_credits"
msgstr "translator_credits"

# E: 번역자 써야 함
#: test.c:7
msgid "translator_credits"
msgstr "번역한_분들"

#: test.c:7
msgid "translator_credits"
msgstr "류창우 <cwryu@debian.org>"

# E: 번역자 써야 함, 이렇게 번역하면 진짜 바보
#: test.c:7
msgid "translator_credits-PLEASE_ADD_YOURSELF_HERE"
msgstr "번역한_분들-여기에_자기자신을_추가"
