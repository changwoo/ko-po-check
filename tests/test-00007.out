test-00007.po:18: GNOME에서 "translator_credits"에는 번역자들의 이름을 써야 합니다
test-00007.po:23: GNOME에서 "translator_credits"에는 번역자들의 이름을 써야 합니다
test-00007.po:32: GNOME에서 "translator_credits"에는 번역자들의 이름을 써야 합니다
test-00007.po:32: 추가: 그놈 데스크탑에서 "add"은(는) "더하기"(이)라고 번역
